packages/cli/src/environments/sourceControl/sourceControlImport.service.ee.ts

```diff
@@ -508,6 +508,7 @@ export class SourceControlImportService {
 					projectRepository.create({
 						id: owner.teamId,
 						name: owner.teamName,
+						type: 'team',
```

Comment:
https://github.com/n8n-io/n8n/blob/80445cd17874542ea5bf67cbcd3c1780f0bf5c82/packages/cli/src/databases/migrations/common/1714133768519-CreateProject.ts#L51-L51

Could we add `notNull` here? Or would that break something?

Reply:
![image](https://github.com/n8n-io/n8n/assets/927609/268a0846-b08c-4e81-9116-d32872ba20d2)

That would have caught it and all the other tests are passing, but I'm not sure if there is reason for that field to be nullable. In the TypeORM schema we already mark it as non-nullable.